### DISCUSSION TITLE
Is it possible to change 'short' during url creation

### DISCUSSION THREAD

**Discussion by satyam147** [ash-jc-allen/short-url#74] [Q&A] [answered]
I want to know whether we can change hardcoded 'short' while creating.

**Comment by ash-jc-allen** [comment, marked as answer]
Hey! At the moment, there's no functionality added for this to be changed. But it seems like something that quite a few people have asked about in the past.

I've not really got any plans to do this at any point myself. But, I'd be more than happy to look at any PRs if it's something you maybe wanted to add?

**Reply by satyam147** [reply]
Ya, I will get back to you with a PR.

**Reply by TechTailor** [reply]
@satyam147 any plans on the PR or should I submit one myself?

Right now I am just customizing my personal build to change /short to /s or remove it entirely if needed.

-TT
